packages/itmat-ui-react/src/components/lxd/lxd.instance.create.tsx

```diff
@@ -0,0 +1,134 @@
+import React, { useState } from 'react';
+import { Button, Form, Input, message, Select } from 'antd';
+import axios from 'axios';
+
+const { Option } = Select;
+
+const pollOperation = async (operationUrl: string): Promise<void> => {
+    return new Promise<void>((resolve, reject) => {
+        const interval = setInterval(async () => {
+            try {
+                const opResponse = await axios.get(`http://localhost:3333${operationUrl}`);
+                const opData = opResponse.data;
+
+                if (opData.metadata.status === 'Success') {
+                    clearInterval(interval);
+                    resolve(); // No value is being passed here
+                } else if (opData.metadata.status === 'Failure') {
+                    clearInterval(interval);
+                    reject(new Error('Operation failed'));
+                }
+                // Add more conditions as necessary based on the LXD API
+            } catch (error) {
+                clearInterval(interval);
+                reject(error);
+            }
+        }, 2000); // Poll every 2 seconds. Adjust timing as needed.
+    });
+};
+
+
+const CreateInstance = ({ onInstanceCreated }) => {
+    const [isCreating, setIsCreating] = useState(false);
+
+    // const backendEndpoint = 'http://localhost:3333/lxd/1.0/instances';
+    const backendEndpoint = 'http://localhost:3333/api/lxd';
```

Comment:
This is obviously very false, there is already a connection manager dependant on the configuration to get these values. This must be fixed

Reply:
this module has been removed.